[Security] 8.17.9 release notes (#6913)
elastic/security-docs: docs/release-notes/8.17.asciidoc

[[release-notes-header-8.17.0]]
== 8.17

[discrete]
[[release-notes-8.17.9]]
=== 8.17.9

[discrete]
[[enhancements-8.17.9]]
==== Enhancements
* Shortens the time it takes to recover from a `DEGRADED` status caused by {elastic-agent} communication issues.

[discrete]
[[bug-fixes-8.17.9]]
==== Fixes

> * Fixes an issue in {elastic-defend} that may result in bugchecks (BSODs) on Windows systems with a very high volume of network connections. This issue has only been observed on Windows Server.

Reviewer: ```suggestion
* Fixes an issue in {elastic-defend} that sometimes caused bugchecks (BSODs) on Windows Server systems that had a very high volume of network connections. 
```

> * Fixes a bug where Linux endpoint network events would fail to load if IPv6 is not supported by the system.

Reviewer: ```suggestion
* Fixes a bug where Linux endpoint network events would fail to load if IPv6 was not supported by the system.
```
